### DISCUSSION TITLE
Completely new, just got this a day ago and trying to figure out what is happening

### DISCUSSION THREAD

**Discussion by BirdoGunner** [rejetto/hfs#723] [Q&A]
Hi, I was trying to port forward and use HTTPs, when I try to access it on my laptop, it didn't work. When I tested the connection outside it says, error 503. I've tried diagnosing it but I am so stumped to the point that I've exhausted my options and resorted to asking here
![image](https://github.com/user-attachments/assets/06005b05-138c-4bb5-a34e-e1918e7f87d5)

**Comment by rejetto** [comment]
that 503 is because the service is broken.
give me some time and i'll fix it.
thanks for reporting!

**Comment by rejetto** [comment]
ok, i've fixed it.
try again.
that tool should also give advice in case it fails.

**Reply by BirdoGunner** [reply]
Got it, lemme check again

**Reply by BirdoGunner** [reply]
My connection isnt marked as secure even though i did a self signed one, is it because its self signed or should i get an actual one

**Comment by rejetto** [comment]
yes, it's normal  
<img width="416" alt="image" src="https://github.com/user-attachments/assets/4c3ae435-1601-4a3b-896e-f55c557e017d">
when you clicked self-signed there was another link with an explanation, the "won't be perfect".
That link brings you to https://github.com/rejetto/hfs/wiki/HTTPS
if you have a domain you can request a proper certificate.
